<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVideoStitcherDemo</class>
 <widget class="QWidget" name="QtVideoStitcherDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtVideoStitcherDemo</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>打开视频文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>841</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>视频1：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>视频2：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>主控视频：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>视频1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>视频2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>融合视频</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SURF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SIFT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AKAZE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ORB</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Finder：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Match_conf：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>229</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>281</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Conf_thresh：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NoBundleAdjuster</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BundleAdjusterReproj</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BundleAdjusterRay</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BundleAdjusterAffinePartial</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjuster:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Warper_creator:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AffineWarper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PlaneWarper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CylindricalWarper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SphericalWarper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FisheyeWarper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>StereographicWarper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CompressedRectilinearWarper(参数高)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CompressedRectilinearWarper(参数低)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CompressedRectilinearPortraitWarper(参数高)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CompressedRectilinearPortraitWarper(参数低)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PaniniWarper(参数高)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PaniniWarper(参数低)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PaniniPortraitWarper(参数高)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PaniniPortraitWarper(参数低)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MercatorWarper</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>保存融合图像</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>569</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Brightness adjustment:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>680</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrast adjustment:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>730</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>流畅</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高清</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>超清</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清晰度选择：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Progress info:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>目标检测开关</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QtVideoStitcherDemo</receiver>
   <slot>changeMainVideoShowMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QtVideoStitcherDemo</receiver>
   <slot>setScreenShootFlag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtVideoStitcherDemo</receiver>
   <slot>setBrightnessAdjustment(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtVideoStitcherDemo</receiver>
   <slot>setContrastAdjustment(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>679</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>724</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_5</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QtVideoStitcherDemo</receiver>
   <slot>setDefinition(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>QtVideoStitcherDemo</receiver>
   <slot>setmovingDetectionFlag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>779</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtVideoStitcherDemo</receiver>
   <slot>videoOpenprocess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>videoOpenprocess()</slot>
  <slot>changeMainVideoShowMode(int)</slot>
  <slot>setScreenShootFlag()</slot>
  <slot>setBrightnessAdjustment(int,int)</slot>
  <slot>setContrastAdjustment(int,int)</slot>
  <slot>setContrastAdjustment(int)</slot>
  <slot>setBrightnessAdjustment(int)</slot>
  <slot>setDefinition(int)</slot>
  <slot>setmovingDetectionFlag()</slot>
 </slots>
</ui>
